packages/vscode-extension/e2e_tests/tests/commandHandlers.spec.ts

```diff
@@ -88,4 +94,78 @@ suite('Command handlers spec', () => {
 
     sandbox.assert.notCalled(sendNotificationSpy);
   });
+
+  test('Switching a database should notify the language server', async () => {
+    const sendNotificationSpy = sandbox.spy(
+      mockLanguageClient,
+      'sendNotification',
+    );
+    getActiveConnectionStub = sandbox
+      .stub(connection, 'getActiveConnection')
+      .returns(getMockConnection(true));
+
+    await switchToDatabase({
+      type: 'database',
+      label: '',
+      description: 'active',
+      collapsibleState: 0,
+      key: 'schemas',
+    });
+
+    sandbox.assert.calledOnceWithExactly(
+      sendNotificationSpy,
+      'connectionUpdated',
+      {
+        trace: { server: 'off' },
+        connect: true,
+        connectURL: 'neo4j://localhost:7687',
+        database: 'schemas',
+        user: 'neo4j',
+        password: null,
+      },
+    );
```

Comment:
Aren't we missing testing that database is an `activeDatabase` now?

Reply:
we've got a test in the `connectionTreeDataProvider` spec that tests for this (`An active Connection should show the specified database as active`) if this is what you were thinking?